code/src/CoreTemplateStudio/CoreTemplateStudio.Core/Helpers/Fs.cs

```diff
@@ -16,65 +14,27 @@ namespace Microsoft.Templates.Core.Helpers
 {
     public static class Fs
     {
-        public static void EnsureFolder(string folder)
+        public static void EnsureFolderExists(string folder)
         {
-            if (!Directory.Exists(folder))
-            {
-                Directory.CreateDirectory(folder);
-            }
-        }
-
-        public static void CopyRecursive(string sourceDir, string targetDir, bool overwrite = false)
```

Comment:
CopyRecursive is used in WinTS in the VisualComparisonTests, we should keep it in the FSHelper for now.